### PR TITLE
Fix #71 Support component of class.

### LINKED ISSUE #71: can't use es6 class component
Hello,

I found a problem when use this plugin with ES6 class component ( https://mithril.js.org/components.html#es6-classes ).

node raises error `TypeError: Cannot call a class as a function`, because mithril-node-renderer calls es6 class as function in this.
https://github.com/MithrilJS/mithril-node-render/blob/master/index.js#L175

Best regards,

### PR DESCRIPTION
**Pull request by alfa-jpn**
Hi,

Fixes https://github.com/MithrilJS/mithril-node-render/issues/71.
Support component type of class.
(https://mithril.js.org/components.html#es6-classes)

best regards,

### CODE REVIEW

**Comment by StephanHoyer** [comment]
Hi @alfa-jpn 

thanks, great work!

Can you please add a test that proves that state management works as expected and the constructor is not called again in redraw. Same goes for lifecycle hooks and embedded usage. I think it might work as expected, I only want to be sure it is.

Thanks in advance.
Stephan

**Comment by alfa-jpn** [comment]
@StephanHoyer 

Thanks for reply.

Sorry, 
Because I don't know much about inside of mithril,
I had no idea how to test status management and redrawing.

So, I added only lifecycle and embedded tests.

best regards,

File: index.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ function * createChildrenContent (view, options, hooks) {
 
 function * render (view, attrs, options) {
   options = options || {}
-  if (view.view) { // root component
+  if (view.view || isClass(view)) { // root component
```
Comment (alfa-jpn) [inline index.js]: for root
https://github.com/MithrilJS/mithril-node-render/pull/72/commits/441385e89a6486e3a52a2545b679dd57881cc0c1#diff-1dd241c4cd3fd1dd89c570cee98b79ddR145

**Comment by StephanHoyer** [comment]
LGTM! Awesome work.

**Comment by StephanHoyer** [comment]
Just released v2.2.0 with your changes.

**Pull request merged**